chessington/engine/pieces.py

```diff
@@ -64,18 +108,15 @@ def check_attack(self, board, square: Square) -> bool:
 	def get_available_moves(self, board) -> List[Square]:
 		current_square = board.find_piece(self)
 		available_moves = []
+		direction = 1
 
 		if self.player == Player.BLACK:
-			square_in_front = Square.at(current_square.row - 1, current_square.col)
-			two_squares_in_front = Square.at(current_square.row - 2, current_square.col)
-			attack_left = Square.at(current_square.row - 1, current_square.col - 1)
-			attack_right = Square.at(current_square.row - 1, current_square.col + 1)
+			direction = -1
```

Comment:
I think you can use a ternary operator here: https://www.geeksforgeeks.org/ternary-operator-in-python/